Migration 1804 v2

ninafiona opened the pull request:
* migration to ubuntu 18.04, ros melodic, no ocado code

Commit:
add migrating changes

Commit:
removing Ocado code based ifco detection

ninafiona committed:
added yaml and uncommented rbo detection in CMakeLists

ninafiona committed:
vision cfg added and yaml modified

ninafiona committed:
readme folder

ninafiona committed:
readme updated

ninafiona committed:
Update README.md

ninafiona committed:
Merge branch 'master' into migration_1804_v2

ninafiona committed:
merged with master

ninafiona committed:
mergefix

ninafiona committed:
merged

zweistein commented on README.md:
````diff
@@ -105,21 +175,23 @@ Launch camera or play bag file
+rosrun rviz rviz -d `rospack find ecto_rbo_yaml`/cfg/ifco_example.rviz
 ```
 
 Execute vision
````
The Execution lacks some instructions: where is the bag, how to lunch it, which topics to publish, and what is the expected outcome (e.g., a picture of RVIZ visualisation). 
The simple 'demo_vision.yaml` should be also added.

zweistein committed:
Delete bla

remove tmp debug file

zweistein commented:
No build items should be added to the rep, like ecto_rbo_pcl/doc/build/.doctrees/cells.doctree please remove all of them

ninafiona committed:
deleted build doc files

ninafiona committed:
pull
Merge branch 'migration_1804_v2' of https://github.com/SoMa-Project/vision into migration_1804_v2

ninafiona committed:
Update README.md

ninafiona committed:
adjusted the ec_graph_service option

ninafiona committed:
pulled from origin
Merge branch 'migration_1804_v2' of https://github.com/SoMa-Project/vision into migration_1804_v2

ninafiona committed:
readme record bag file

ninafiona committed:
Add files via upload

ninafiona committed:
Update README.md

ninafiona committed:
Update README.md

ninafiona committed:
Update README.md

ninafiona committed:
Update IfcoDetection.cpp

ninafiona committed:
Update README.md

ninafiona committed:
req adjusted

ninafiona committed:
delete merge message
updated the static transformer code due to an error

ninafiona committed:
quick fix merge error

ninafiona committed:
merge from newer version
Merge branch 'migration_1804_v2' of https://github.com/SoMa-Project/vision into migration_1804_v2

zweistein committed:
fix merge issue in commented ifco detection from Ocado

zweistein reviewed (approved):
Functionally tested on WAM in 12.2020
